tests/outbox/test_outbox.py

```diff
@@ -92,3 +94,36 @@ def test_tries_to_send_up_to_three_times(
         outbox.run_once()
 
     assert len(publisher.mock_calls) == 3
+
+
+class TestFiltererWhichFiltersOutEverything:
+    @pytest.fixture()
+    def filterer(self) -> OutboxFiltererStrategy:
+        return lambda event: False
+
+    @pytest.fixture(
+        params=[
+            ("esdb_factory", "esdb_outbox_storage_strategy"),
+            ("sqlite_factory", "sqlite_outbox_storage_strategy"),
+            ("postgres_factory", "postgres_outbox_storage_strategy"),
+        ]
+    )
+    def event_store(self, request: SubRequest) -> EventStore:
```

Comment:
I understand why you needed to create a custom event store, although I think it should be a bit easier and one should be able to pass `filterer` directly to `event_store_factory`. OTOH, I think we miss a nice factory / builder to let assemble the event store. I'll implement it next. For now, tech debt is on me. I'll pay it soon, tho